src/components/AllArticles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api-fetch-articles";
+import { Link } from "react-router-dom";
+
+const AllArticles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then((articlesData) => setArticles(articlesData));
```

Comment:
extract this out to its own .js file in a utils folder for good separation of concerns